molecule/bgp/molecule.yml

```diff
@@ -0,0 +1,41 @@
+# Copyright (c) 2024 VEXXHOST, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+dependency:
+  name: galaxy
+driver:
+  name: default
+  options:
+    managed: False
+    ansible_connection_options:
+      ansible_become: "true"
+      ansible_connection: local
+platforms:
+  - name: instance
+    groups:
+      - controllers
+provisioner:
+  name: ansible
+  config_options:
+    connection:
+      pipelining: true
+  inventory:
+    group_vars:
+      all:
+        kubernetes_version: ${KUBERNETES_VERSION-1.28.13}
+        cilium_helm_values:
+          operator:
+            replicas: 1
+      controllers:
+        keepalived_interface: "{{ ansible_facts['default_ipv4'].interface }}"
+        keepalived_vip: 172.17.0.100
+        keepalived_vrid: 42
```

Comment:
Just an FYI we should get rid of these, because we are using kube_vip everywhere now... these are just transitionary old variables.